### PR TITLE
Ansible ZipShip integration

### PR DESCRIPTION
**Pull request by sabre1041**
Creation of repositories and teams for Ansible ZipShip effort

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -325,6 +328,46 @@ orgs:
         privacy: closed
         repos:
           ansible-snapshot: admin
+      ansible-zipship:
+        description: "Ansible ZipShip"
+        maintainers:
+          - A-Beck
+          - sabre1041
+          - bontreg
+          - brady-s
```
Comment (pabrahamsson) [inline config.yaml:337]:
```suggestion
          - bontreger
          - brady-spiva
```

**Review by pabrahamsson — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (sabre1041) [reply config.yaml]: @pabrahamsson corrected

File: config.yaml [hunk 2]
```diff
@@ -325,6 +328,46 @@ orgs:
         privacy: closed
         repos:
           ansible-snapshot: admin
+      ansible-zipship:
+        description: "Ansible ZipShip"
+        maintainers:
+          - A-Beck
+          - sabre1041
+          - bontreg
```
Comment (pabrahamsson) [inline config.yaml:336]: I you missed this one @sabre1041 
```suggestion
          - bontreger
```

Comment (sabre1041) [reply config.yaml]: 3rd time is the charm

File: config.yaml [hunk 3]
```diff
@@ -325,6 +328,46 @@ orgs:
...
         repos:
           ansible-snapshot: admin
+      ansible-zipship:
+        description: "Ansible ZipShip"
+        maintainers:
+          - A-Beck
+          - sabre1041
+          - bontreger
+          - brady-spiva
+          - jeffcpu
```
Comment (pabrahamsson) [inline config.yaml:338]: Maybe this one too @sabre1041? Sorry I missed it before
```suggestion
          - jeffcpullen
```

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Pull request merged**